x-pack/plugins/apm/public/components/fleet_integration/apm_agents/agent_instructions_accordion.tsx

```diff
@@ -97,96 +107,167 @@ function TutorialConfigAgent({
 }
 
 interface Props {
+  policy: PackagePolicy;
   newPolicy: NewPackagePolicy;
+  onChange: PackagePolicyEditExtensionComponentProps['onChange'];
   agentName: AgentName;
   title: string;
   variantId: string;
   createAgentInstructions: CreateAgentInstructions;
+  AgentRuntimeAttachment?: (props: AgentRuntimeAttachmentProps) => ReactElement;
 }
 
+const StyledEuiAccordion = styled(EuiAccordion)`
+  //This is an alternative fix suggested by the EUI team to fix drag elements inside EuiAccordion
```

Comment:
```suggestion
  // This is an alternative fix suggested by the EUI team to fix drag elements inside EuiAccordion
```

ESLint complains about spaces after comments start, but apparently not in a `styled` block.